xbmc/platform/linux/input/LibInputHandler.cpp

```diff
@@ -116,7 +116,14 @@ void CLibInputHandler::Start()
 
 void CLibInputHandler::Process()
 {
+#if defined(TARGET_FREEBSD)
+  // epoll_create1() from libepoll-shim always fails with 0 and
+  // only supports EPOLL_CLOEXEC, so let's use that to make Kodi/GBM
+  // runnable on FreeBSD.
+  int epollFd = epoll_create1(EPOLL_CLOEXEC);
```

Comment:
Can you just make this the default and remove the ifdeffing? I'll give it a go on linux but I don't see why it wouldn't work.

Reply:
Ok, done.

Reply:
So was it fine with EPOLL_CLOEXEC on Linux too?

Reply:
Seems fine to me. I just tested it. Sorry for the delay. I just moved houses and finally set my workstation up again.